Surf Base Application.xctemplate/Podfile

```
@@ -1,18 +1,30 @@
 platform :ios, '10.0'
```

Здесь наверное уже 11 стоит указать?
```suggestion
platform :ios, '11.0'
```